README.md

```diff
@@ -400,6 +401,14 @@ List of podcasts which are helpful for software engineers/programmers.
   - <b>Frequency</b>  : Once every week
 
 
+## Vue.js
```

Comment:
Do you want to merge this under Web Development or Javascript category ?

Reply:
I thought about that for a second, but then I noticed Angular also has a dedicated section.
I've joined them into a dedicated JavaScript section.